nessie/models/development_db.py

```diff
@@ -42,12 +84,135 @@ def load():
 
 
 def load_development_data():
-    pass
+    load_student_athletes()
 
 
 def load_schemas():
     """Create DB schema from SQL file."""
     with open(app.config['BASE_DIR'] + '/scripts/db/schema.sql', 'r') as ddlfile:
         ddltext = ddlfile.read()
     db.session().execute(text(ddltext))
-    std_commit()
+    std_commit(allow_test_environment=True)
```

Comment:
@johncrossman Ah, there it is. I was getting flummoxed about why schemas weren't schematizing.